compiler/optimizations/gpuTransforms.cpp

```diff
@@ -92,6 +92,10 @@ static SymExpr* hasOuterVarAccesses(FnSymbol* fn) {
   for_vector(SymExpr, se, ses) {
     if (VarSymbol* var = toVarSymbol(se->symbol())) {
       if (var->defPoint->parentSymbol != fn) {
+        if (var->name == astr("chpl_privateObjects")) {
```

Comment:
Maybe a TODO about adding a flag for this kind of case, in the event we want to skip this for other variables?